tests/test_utilities.py

```diff
@@ -16,3 +19,62 @@ def fill_para_sim_template(msw_folder: Path, path_unsat_dbase: Path) -> None:
 
     with open(msw_folder / "para_sim.inp", mode="w") as f:
         f.write(para_sim_text)
+
+
+def numeric_csvfiles_equal(
+    file1: Path,
+    file2: Path,
+    sep: str,
+    abstol: float,
+    reltol: float,
+) -> bool:
+    df1 = pd.read_csv(
+        file1,
+        sep,
+    )
+    df2 = pd.read_csv(
+        file2,
+        sep,
+    )
+    if df1.shape != df2.shape:
+        print(f"the dataframes in {file1} and {file2} do not have the same shape")
+        return False
+    return numeric_dataframes_equal(df1, df2, abstol, reltol)
+
+
+def numeric_dataframes_equal(
```

Comment:
Wouldn't this function make more sense?
https://pandas.pydata.org/pandas-docs/version/0.22.0/generated/pandas.testing.assert_frame_equal.html#pandas.testing.assert_frame_equal

Reply:
Rather than absolute and relative tolerance, this functio offers a  "check_less_precise" option. Then you choose how many digits you want to compare. I believe that  is not good enough for our purpose.  I prefer being able to implement our own meaning of tolerance

Reply:
Fair enough.